Source/SwiftLintBuiltInRules/Rules/Style/ImplicitGetterRuleExamples.swift

```diff
@@ -30,6 +30,15 @@ struct ImplicitGetterRuleExamples {
                 }
             }
             """),
+        Example("""
+            class Foo {
+                var foo: Int {
+                    @storageRestrictions(initializes: _foo)
```

Comment:
`_foo` should be a property of the class to make it valid Swift, right?

Reply:
Yep, it gets through parsing but it would probably fail during sema. I left it out from the example following suit with the previous example that referred to `_abc`; assumed that was for brevity. Want me to add an explicit `_foo` property here (and maybe an `_abc` to the example right before this one)?

Reply:
Updated this particular instance — thinking about it a bit I'm not sure if the other examples _need_ to be updated considering that `_abc` etc. could conceivably be declared elsewhere (e.g. global computed vars). Seems almost all the examples in this file use `_abc`/`_foo`/... and adding the backing prop could add syntactic noise. That said if you do feel strongly about the change I'd be happy to add the backing props elsewhere.

Reply:
Fair enough. Other examples require some cleanup too, but that's not your job in this PR. Follow-ups are always welcome though.